frontend/src/components/BotList.svelte

```diff
@@ -62,18 +67,45 @@ const subCategories: { [x: string]: string[] } = {
     "Heatseeker",
   ],
 };
```

Comment:
I suggest making some classes for this. When keys are arrays of entries from another 2d array, it is time to add some structure lol.

Reply:
done!